pinot-core/src/test/java/org/apache/pinot/queries/FilteredAggregationsTest.java

```diff
@@ -88,13 +88,12 @@ public void setUp()
 
     buildSegment(FIRST_SEGMENT_NAME);
     buildSegment(SECOND_SEGMENT_NAME);
-    IndexLoadingConfig indexLoadingConfig = new IndexLoadingConfig();
 
     Set<String> invertedIndexCols = new HashSet<>();
     invertedIndexCols.add(INT_COL_NAME);
 
-    indexLoadingConfig.setInvertedIndexColumns(invertedIndexCols);
-    indexLoadingConfig.setRangeIndexColumns(invertedIndexCols);
+    IndexLoadingConfig indexLoadingConfig =
+        new IndexLoadingConfig(null, createTableConfig(invertedIndexCols, invertedIndexCols));
```

Comment:
In production, we always have table config and schema when loading a segment. Let's use the constructor which takes `TableConfig` and `Schema`. You may make the schema as constant (see `ForwardIndexHandlerTest` as an example)

Reply:
Consider inline `createTableConfig()`. It is a little bit confusing putting 2 `invertedIndexCols`